python/caffe/_caffe.cpp

```diff
@@ -275,7 +275,7 @@ BOOST_PYTHON_MODULE(_caffe) {
   bp::class_<vector<bool> >("BoolVec")
     .def(bp::vector_indexing_suite<vector<bool> >());
 
-  import_array();
+  import_array1();
```

Comment:
Is that a boost issue? Surely boost should define an init function with the right type for the Python version? It does seem that  `import_array` does the right thing, which is to return void for 2 and `NULL` for 3. What actually goes wrong with `import_array`? It sounds like boost thinks it's being built for 2...